DOCS.md

```diff
@@ -0,0 +1,45 @@
+# Structure
+
+Guardrails docs are served as a docusaurus site. The docs are compiled from various sources
+
+1. Manually written markdown files in the `docs` directory
+2. Python notebooks in the `docs` directory translated to markdown using nb-docs
+3. Automatically generated python docs from the `guardrails` directory
+
+These sources need to be built and compiled before the site can be served. 
+
+
+## Installation
+
+```bash
+# First, make sure you're in a venv
+python -m venv .venv
+source .venv/bin/activate
+
+# Install poetry
+pip install poetry
```

Comment:
_Technically_, the Poetry docs would suggest running this before creating the virtual environment.  I've never run into any issues doing it this way, but I think it's a "buyer be warned" kind of scenario.